hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -84,28 +84,88 @@ Default: 0x00000001 (with 3 SD boot retries to match the current bootloader beha
 Specify the maximum number of times that the bootloader will retry booting from the SD card.  
 -1 means infinite retries  
 Default: 0  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### NET_BOOT_MAX_RETRIES
 Specify the maximum number of times that the bootloader will retry network boot.  
 -1 means infinite retries  
 Default: 0  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### DHCP_TIMEOUT
 The timeout in milliseconds for the entire DHCP sequence before failing the current iteration.  
 Default: 45000  
 Minimum: 5000  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### DHCP_REQ_TIMEOUT
 The timeout in milliseconds before retrying DHCP DISCOVER or DHCP REQ.  
 Default: 4000  
 Minimum: 500  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_TIMEOUT
 The timeout in milliseconds for an individual file download via TFTP.  
 Default: 15000  
 Minimum: 5000  
+Version: stable/pieeprom-2020-01-17.bin  
 
 ### TFTP_IP
 Optional dotted decimal ip address (e.g. 192.169.1.99) for the TFTP server which overrides the server-ip from the DHCP request.  
 This maybe useful on home networks because tftpd-hpa can be used instead of dnsmasq where broadband router is the DHCP server.
 Default: ""  
+Version: stable/pieeprom-2020-01-17.bin  
+
+### TFTP_PREFIX
+In order to support unique TFTP boot directories for each Pi the bootloader prefixes the filenames with a device specific directory. If neither start4.elf nor start.elf are found in the prefixed directory then the prefix is cleared.
+On earlier models the serial number is used as the prefix, however, on Pi4 the MAC address is no longer generated from the serial number making it difficult to automatically create tftpboot directories on the server by inspecting DHCPDISCOVER packets. To support this the TFTP_PREFIX may be customized to either be the MAC address or a fixed value.
```

Comment:
Small nitpick, but it says here "the TFTP_PREFIX may be customized to either be the MAC address or a fixed value." but in the list below the serial number is also an option? I guess this ought to be changed to "the TFTP_PREFIX may be customized to be the serial number, the MAC address or a fixed value."